gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -279,13 +279,17 @@ public GsonBuilder enableComplexMapKeySerialization() {
   }
 
   /**
-   * Configure Gson to deserialize duplicate map key. By default, Gson throw a {@code JsonSyntaxException} when there are more than
-   * one same key.
+   * Configures Gson to deserialize duplicate map keys. Only the value of last entry with the same key will be used, previous values
+    * will be discarded. By default, Gson throws a {@link JsonSyntaxException} when a key occurs more than once.
```

Comment:
Sorry, looks like I got the indentation wrong.
```suggestion
   * will be discarded. By default, Gson throws a {@link JsonSyntaxException} when a key occurs more than once.
```